qml/Panel/Panel.qml

```diff
@@ -48,6 +48,9 @@ Item {
     property bool greeterShown: false
     property bool hasKeyboard: false
 
+    // Whether our expanded menus should take up the full width of the panel
+    property bool partialWidth: width > units.gu(60)
```

Comment:
This will become false when `width` is 1 pixel above 60 GUs. Is this intentional? Or, what is the correct approach here? Does it matter?

Reply:
There's really no other way to make breakpoints. There will always be a one-pixel where you can cause the layout to change.

Reply:
I mean, is this supposed to be `>=` instead of `>`? In other words, which one-pixel spot should we consider?

Reply:
I'll use `>=`.